bridge/server/views/logout.pug

```diff
@@ -0,0 +1,18 @@
+html
+  include common.pug
+
+  head
+    link(rel="stylesheet", href="static/error.v1.css")
+    title="Logout"
+  body
+    div.container
+
+      div.image
+        img(src="static/logo.png", alt="Logo")
```

Comment:
We need to override this file in case that custom Look-and-Feel is configured.

Reply:
Changed it to take the `logo_inverted.png` from our `branding` folder